frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/EnvironmentHeader.tsx

```diff
@@ -17,9 +17,6 @@ const StyledAccordionSummary = styled(AccordionSummary, {
     padding: theme.spacing(0.5, 3, 0.5, 2),
     display: 'flex',
     alignItems: 'center',
-    [theme.breakpoints.down('sm')]: {
-        fontWeight: 'bold',
-    },
```

Comment:
This line was marked as new in the recent changes according to my editor. @Tymek Is this intentional? I don't see why we would want to bold it on smaller screens?

Reply:
Probably was intentional at the time. Requirements change :P